aptos-move/aptos-vm-types/src/resource_group_adapter.rs

```diff
@@ -175,10 +177,9 @@ impl TResourceGroupView for ResourceGroupAdapter<'_> {
         resource_tag: &Self::ResourceTag,
         maybe_layout: Option<&MoveTypeLayout>,
     ) -> anyhow::Result<Option<Bytes>> {
-        if let Some(group_view) = self.maybe_resource_group_view {
+        if let Some(group_view) = &self.maybe_resource_group_view {
```

Comment:
Isn't this already a reference?

Reply:
`self.maybe_resource_group_view` was Option<&....>.

Reply:
but it's not changed in the PR, still &?
basically, if you remove &, it should still work I think since the definition (as of right now) isn't affected?
(&& collapses to & though)

Reply:
Done.